### PR TITLE
feat(testplan): add troubleshooting section in testplan

### PR DESCRIPTION
**Pull request by harshshekhar15**
Signed-off-by: Harsh Shekhar <harsh.shekhar@mayadata.io>

This PR intends to do the following:
- Add `Troubleshooting` section in testplan site.
- Add Rancher troubleshooting docs.

#### Versions:

```
$ Kubernetes version
$ Kubernetes platform
$ kubectl version

```

### CODE REVIEW

File: testplan/docs/troubleshooting/rancher.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
...ric, unable to get CPU for container "weave-plugins" in pod kube-system/weave-net-nbrz7 on node "10.53.1.12", discarding data: missing cpu usage metric, unable to get CPU for container "weave" in pod kube-system/weave-net-nbrz7 on node "10.53.1.12", discarding data: missing cpu usage metric, unable to get CPU for container "weave-npc" in pod kube-system/weave-net-nbrz7 on node "10.53.1.12", discarding data: missing cpu usage metric, unable to get CPU for container "nginx-ingress-controller" in pod ingress-nginx/nginx-ingress-controller-2xzvq on node "10.53.1.12", discarding data: missing cpu usage metric, unable to get CPU for container "agent" in pod cattle-system/cattle-node-agent-kgb2q on node "10.53.1.12", discarding data: missing cpu usage metric, unable to get CPU for container "autoscaler" in pod kube-system/coredns-autoscaler-65bfc8d47d-4zj2n on node "10.53.1.12", discarding data: missing cpu usage metric], unable to fully scrape metrics from source kubelet_summary:oep-rancher-node1: unable to fetch metrics from Kubelet oep-rancher-node1 (10.53.1.11): Get https://10.53.1.11:10250/stats/summary?only_cpu_and_memory=true: dial tcp 10.53.1.11:10250: connect: connection refused]
```
Comment (AmitKumarDas) [inline testplan/docs/troubleshooting/rancher.md:38]: Can this be indented? We dont want to scroll horizontally.

File: testplan/docs/troubleshooting/rancher.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+## Issues
+
+### Issue1: Openebs installation failing on cluster2
+
+#### What is the issue?
+
+In the Rancher pipeline, the installation of OpenEBS via Kubera OnPrem Director was stuck. On investigating more we found out that openebs-upgrade and cstorpoolauto pod present in maya-system namespace was showing the following error -
+
+```
+E0730 05:58:27.664789       1 discovery.go:184] API resource discovery failed: unable to retrieve the complete list of server APIs: metrics.k8s.io/v1beta1: the server is currently unable to handle the request
```
Comment (AmitKumarDas) [inline testplan/docs/troubleshooting/rancher.md:16]: Indent please

File: testplan/docs/troubleshooting/rancher.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+sidebar_label: Rancher
+---
+
+## Issues
+
+### Issue1: Openebs installation failing on cluster2
+
+#### What is the issue?
+
+In the Rancher pipeline, the installation of OpenEBS via Kubera OnPrem Director was stuck. On investigating more we found out that openebs-upgrade and cstorpoolauto pod present in maya-system namespace was showing the following error -
```
Comment (gprasath) [inline testplan/docs/troubleshooting/rancher.md:13]: Timestamp mismatch between master and nodes component was the actual root cause for metrics-server kube-apiservice to not perform. Those details are to be included for better understanding in future

Comment (harshshekhar15) [reply testplan/docs/troubleshooting/rancher.md:13]: Good point @gprasath. I'll add that.

File: testplan/docs/troubleshooting/rancher.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+`
+E0730 05:58:27.664789       1 discovery.go:184] API resource discovery failed: unable to retrieve the complete list of server APIs: metrics.k8s.io/v1beta1: the server is currently unable to handle the request
+`
+
+On running kubectl get apiservices we got the below output -
```
Comment (gprasath) [inline testplan/docs/troubleshooting/rancher.md:19]: Can you differentiate the kubectl command from regular text

**Pull request merged**